### DISCUSSION TITLE
pi

### DISCUSSION THREAD

**Discussion by fahriromadan** [community/community#146742] [Other Feature Feedback, Questions, & Ideas]

**Comment by mecodeatlas** [comment]
Hi @fahriromadan, we’re glad you’re here 😀

Thanks for sharing your website with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
